cli/oyster-cvm/src/commands/doctor.rs

```diff
@@ -0,0 +1,28 @@
+use crate::types::Dependency;
+use anyhow::Result;
+use log::{error, info};
+use std::process::Command;
+
+pub fn run_doctor() -> Result<()> {
+    check_dependency(Dependency::Docker)?;
+    check_dependency(Dependency::Nix)?;
```

Comment:
even if the Docker check fails, let the Nix check run. And return the complete report at end.

Reply:
done